### ISSUE TITLE
Blackscreen Video not playing

### ISSUE THREAD

**Issue by sainiaj** [shbatm/MMM-RTSPStream#45]
Hi Everyone,
I am able to play the Sricam SP012 P2P HD IP Camera in rasbian vlc player using the url below

rtsp://192.168.1.212:554/onvif2 - Low quality 
rtsp://192.168.1.212:554/onvif1 - HD

however i am just getting a white outline box no stream on the MagicMirror, can someone please advise me what am i doing wrong.I have tried everything to my knowledge 

I have changed the protocol to UDP (however still no luck)
below is the message from tiny cam android APK to configure camera and find network details hope this helps

Parameters used by tinyCam Monitor app to access camera "http://192.168.1.212:80" Name: Sricam (192.168.1.212) Vendor: Sricam Model: SP012 Hostname: 192.168.1.212 Web port: 80 Protocol: RTSP over UDP RTSP port: 554 HTTPS: no Username: 938310 Password: ----- tinyCam Monitor - complete solution for mobile surveillance for Android https://tinycammonitor.com

#####config.js#####
{
module: "MMM-RTSPStream",
position: "middle_center",
config: {
autoStart: true,
rotateStreams: false,
moduleWidth: 640,
moduleHeight: 320,
localPlayer: 'omxplayer',
remotePlayer: 'none',
showSnapWhenPaused: false,
remoteSnaps: false,
stream1: {
name: 'FRONT',
url: "rtsp://192.168.1.212:554/onvif1",
frameRate: undefined,
width: 640,
height: 320,
}
},
},

#####PM2 Logs#####

Starting stream stream1 with args: [
    "--avdict",
    "rtsp_transport:tcp",
    "--win",
    "643, 368, 1283, 688",
    "--live",
    "--video_queue",
    "4",
    "--fps",
    "30",
    "rtsp://192.168.1.212:554/onvif1"
]
Starting PM2 for omx_stream1
PM2 started for omx_stream1
Stopping stream stream1
Stopping PM2 process: omx_stream1
Starting stream stream1 with args: [
    "--avdict",
    "rtsp_transport:tcp",
    "--win",
    "643, 368, 1283, 688",
    "--live",
    "--video_queue",
    "4",
    "--fps",
    "30",
    "rtsp://192.168.1.212:554/onvif1"
]
Starting PM2 for omx_stream1
PM2 started for omx_stream1
Stopping stream stream1
Stopping PM2 process: omx_stream1
Starting stream stream1 with args: [
    "--avdict",
    "rtsp_transport:tcp",
    "--win",
    "643, 368, 1283, 688",
    "--live",
    "--video_queue",
    "4",
    "--fps",
    "30",
    "rtsp://192.168.1.212:554/onvif1"
]
Starting PM2 for omx_stream1
PM2 started for omx_stream1
Stopping stream stream1
Stopping PM2 process: omx_stream1
Starting stream stream1 with args: [
    "--avdict",
    "rtsp_transport:tcp",
    "--win",
    "643, 368, 1283, 688",
    "--live",
    "--video_queue",
    "4",
    "--fps",
    "30",
    "rtsp://192.168.1.212:554/onvif1"
]
Starting PM2 for omx_stream1
PM2 started for omx_stream1

**Comment by sainiaj** [comment]
pi@raspberrypi:~/.pm2/logs $ cat omx-stream1-error.log
pi@raspberrypi:~/.pm2/logs $


#####################
I also noticed this

┌─────────────┬────┬─────────┬──────┬──────┬─────────┬─────────┬────────┬─────┬──────────┬──────┬──────────┐
│ App name    │ id │ version │ mode │ pid  │ status  │ restart │ uptime │ cpu │ mem      │ user │ watching │
├─────────────┼────┼─────────┼──────┼──────┼─────────┼─────────┼────────┼─────┼──────────┼──────┼──────────┤
│ MagicMirror │ 0  │ 2.7.1   │ fork │ 4408 │ online  │ 2       │ 8m     │ 0%  │ 2.5 MB   │ pi   │ enabled  │
│ omx_stream1 │ 1  │ N/A     │ fork │ 0    │ stopped │ 64      │ 0      │ 0%  │ 0 B      │ pi   │ disabled │
└─────────────┴────┴─────────┴──────┴──────┴─────────┴─────────┴────────┴─────┴──────────┴──────┴──────────┘
 Use `pm2 show <id|name>` to get more details about an app
pi@raspberrypi:~/.pm2/logs $ pm2 start omx_Stream1
[PM2][ERROR] script not found : /home/pi/.pm2/logs/omx_Stream1
script not found : /home/pi/.pm2/logs/omx_Stream1
┌─────────────┬────┬─────────┬──────┬──────┬─────────┬─────────┬────────┬─────┬──────────┬──────┬──────────┐
│ App name    │ id │ version │ mode │ pid  │ status  │ restart │ uptime │ cpu │ mem      │ user │ watching │
├─────────────┼────┼─────────┼──────┼──────┼─────────┼─────────┼────────┼─────┼──────────┼──────┼──────────┤
│ MagicMirror │ 0  │ 2.7.1   │ fork │ 4408 │ online  │ 2       │ 9m     │ 0%  │ 2.5 MB   │ pi   │ enabled  │
│ omx_stream1 │ 1  │ N/A     │ fork │ 0    │ stopped │ 64      │ 0      │ 0%  │ 0 B      │ pi   │ disabled │
└─────────────┴────┴─────────┴──────┴──────┴─────────┴─────────┴────────┴─────┴──────────┴──────┴──────────┘
 Use `pm2 show <id|name>` to get more details about an app
pi@raspberrypi:~/.pm2/logs $

**Comment by Anonym-tsk** [comment]
Same issue

**Comment by pooyasa** [comment]
same problem

**Comment by TAGinside** [comment]
Hello,

I have the same problem too.

-My camera stream was in 25fps so I set the frameRate option to 25 but that doesn't change anything in the log it still grows at 30fps.
-Suddenly I did the opposite, I set the stream to 30 fps and that doesn't change either.

I also try to put several resolutions and still nothing.

I would like to point out that on VLC it works without any problem with the sound

Translated with www.DeepL.com/Translator
